app/src/main/res/layout-sw600dp-port/onboarding_slide.xml

```diff
@@ -20,7 +20,7 @@
       android:layout_height="0dp"
       android:contentDescription="@{viewModel.contentDescription}"
       android:src="@{viewModel.slideImage}"
-      app:layout_constraintDimensionRatio="5:4"
+      app:layout_constraintDimensionRatio="H, 384:307"
```

Comment:
This ratio should be `384:295`

Reply:
Also, which image is exactly getting used here? Because I am not able to find any onboarding image which has a ratio like this in drawable folder.
If there is no image like that, then in that case we should ask Chantel to provide such image or else change the ratio as per the image that we are using here.
I personally thing that we should change the ratio to `10:9` which is similar to `ic_portrait_onboarding_0`

Reply:
I'm using `ic_portrait_onboarding_#`, I didn't use new images for the tablet as these images are enough from my point of view.

Reply:
> 
> 
> This ratio should be `384:295`

Done

Reply:
Then in that case the ratio should be 10:9 because image follows that ratio.

Reply:
> 
> 
> Then in that case the ratio should be 10:9 because image follows that ratio.

10:9 results in a white space on the top of the image:
![screenshot-2020-05-31_13 55 40 029](https://user-images.githubusercontent.com/8533363/83351723-c8324f80-a346-11ea-8771-dcf35bbf81b9.png)

5:4 is perfect and it was used [here](https://github.com/oppia/oppia-android/blob/3ee4a28af5cdcb8617d02c872a08d7ce55e136b6/app/src/main/res/layout/onboarding_slide.xml#L23) in the normal portrait devices, I don't know why I have changed that.  

![screenshot-2020-05-31_13 59 43 984](https://user-images.githubusercontent.com/8533363/83351768-1b0c0700-a347-11ea-8a35-d0000f82a73b.png)

![screenshot-2020-05-31_13 59 53 416](https://user-images.githubusercontent.com/8533363/83351766-16dfe980-a347-11ea-8ba7-e5dccdaa70a6.png)

I have pushed the new commits with the ratio 5:4 and removed `android:scaleType="fitXY"` as it is redundunt, PTAL.

Reply:
@rt4914 I have updated the button height to be 54dp as in the mocks, I'll take a screenshot and edit this comment